lib/puma/plugin/systemd.rb

```diff
@@ -10,7 +10,7 @@
 Puma::Plugin.create do
   def start(launcher)
     begin
-      require 'sd_notify'
+      require_relative '../sd_notify'
     rescue LoadError
       launcher.log_writer.log "Systemd integration failed. It looks like you're trying to use systemd notify but don't have sd_notify gem installed"
```

Comment:
Now that it's vendored (https://github.com/puma/puma/commit/6d8b728b42a61bcf3c1e4c698c9165a45e6071e8), I don't think there's a chance that it will fail to load. Certainly the message is wrong now since the `sd_notify` gem is no longer needed.